plugins/arcanist/README.md

```diff
@@ -27,3 +28,13 @@ plugins=(... arcanist)
 | arli    | `arc lint`                         |
 | arls    | `arc list`                         |
 | arpa    | `arc patch`                        |
+
+## Functions
+
+The following functions make copy pasting revision ids from the URL bar of your browser easier as they allow for copy pasting the whole URL.
+E.g., `ardu` accepts `https://arcanist-url.com/<REVISION>` as well as `<REVISION>`.
+
+| Function                  | Command                                  |
+| ------------------------- | ---------------------------------------- |
+| ardu [URL or revision_id] | `arc diff --update` [URL or revision_id] |
+| arpa [URL or revision_id] | `arc patch` [URL or revision_id]         |
```

Comment:
The command part gets a revision only, no URL.

Reply:
✅